zds/forum/models.py

```diff
@@ -228,6 +228,10 @@ def meta_description(self):
             return first_post.text
         return Topic.__remove_greetings(first_post)[: settings.ZDS_APP["forum"]["description_size"]]
 
+    @property
+    def update(self):
```

Comment:
`Topic.last_update` me parait plus compréhensible que `Topic.update` mais c'est simplement une suggestion. :)
```suggestion
    def last_update(self):
```

Reply:
Bonne idée, adoptée